### PR TITLE
Convert stack to composer

### PR DESCRIPTION
**Pull request by andypost**
- removed theme
- added useful commands
- goal `clean` excluded from make

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,28 @@
 .env
-src/docker/docker-compose.override.yml
+docker-compose.override.yml
```
Comment (duozersk) [inline .gitignore]: Seems like this one should be docker/docker-compose.override.yml

Comment (andypost) [inline .gitignore]: no, they are global search pattern https://www.atlassian.com/git/tutorials/gitignore

Comment (duozersk) [inline .gitignore]: OK, but should it really be a global search pattern... do we expect docker-compose.override.yml file somewhere except docker dir?

Comment (andypost) [inline .gitignore]: no, 2 files is enough, one common + override

Comment (duozersk) [inline .gitignore]: Then it really should be docker/docker-compose.override.yml here in .gitignore

Comment (andypost) [inline .gitignore]: makes sense, fixed

**Pull request merged**